app/serializers/page_serializer/statutory_instruments_show_page_serializer.rb

```diff
@@ -22,8 +22,19 @@ def title
       @statutory_instrument.try(:statutoryInstrumentPaperName)
     end
 
+    def title_context
+      paper_prefix  = @statutory_instrument.try(:statutoryInstrumentPaperPrefix)
+      paper_year    = @statutory_instrument.try(:statutoryInstrumentPaperYear)
+      paper_number  = @statutory_instrument.try(:statutoryInstrumentPaperNumber)
+
+      context_content = "#{paper_prefix} #{paper_year}"
```

Comment:
What happens if there is no year?

What happens if there is no prefix?

Reply:
Hi Matt, I've set the `context_content` variable to start as an empty string first and to not render it if it turns out to be empty. Let me know your thoughts.